controlplane/kubeadm/internal/workload_cluster.go

```diff
@@ -60,8 +60,17 @@ const (
 )
 
 var (
+	// Starting from v1.22.0 kubeadm dropped usage of the ClusterStatus entry from the kubeadm-config ConfigMap
+	// so it isn't necessary anymore to remove API endpoints for control plane nodes after deletion.
+	// NOTE: This assume kubeadm version equals to Kubernetes version.
+	minKubernetesVersionWithoutClusterStatus = semver.MustParse("1.22.0")
+
+	// Starting from v1.21.0 kubeadm defaults to systemdCGroup driver, as well as images built with ImageBuilder,
+	// so it is necessary to mutate the kubelet-config-xx ConfigMap.
+	// NOTE: This assume kubeadm version equals to Kubernetes version.
 	minVerKubeletSystemdDriver = semver.MustParse("1.21.0")
-	ErrControlPlaneMinNodes    = errors.New("cluster has fewer than 2 control plane nodes; removing an etcd member is not supported")
+
+	ErrControlPlaneMinNodes = errors.New("cluster has fewer than 2 control plane nodes; removing an etcd member is not supported")
```

Comment:
Add a comment

Reply:
Done (actually you did it while fixing lint errors 😉 )